flink-scala/src/main/java/org/apache/flink/api/scala/typeutils/TraversableSerializerConfigSnapshot.java

```diff
@@ -31,6 +33,7 @@
  * allow calling different base class constructors from subclasses, while we need that
  * for the default empty constructor.
```

Comment:
nit: add `@deprecated` message in Javadoc to redirect to `TraversableSerializerSnapshot`